hadoop-hdfs-project/hadoop-hdfs-rbf/src/main/java/org/apache/hadoop/hdfs/server/federation/metrics/RBFMetrics.java

```diff
@@ -50,6 +50,8 @@
 import javax.management.ObjectName;
 import javax.management.StandardMBean;
 
+import com.fasterxml.jackson.core.JsonProcessingException;
```

Comment:
What's our take on using fasterxml in HDFS?
Is that the common approach?

Reply:
@goiri I had to use fasterxml to fix the main issue - to return a string, I think alternatively we can also use StringBuilder/StringJoiner if using fasterxml is not the desired way here